### PR TITLE
[#1743] Return proper status code for unauthorized access

### LINKED ISSUE #1743: Spring security return proper status code
_see https://github.com/airyhq/airy/pull/1737#discussion_r626812829_

- [x] Make Spring security middleware return the right status code
- [x] Remove redirect detection in httplib

### PR DESCRIPTION
**Pull request by chrismatix**
resolves #1743

### CODE REVIEW

File: backend/api/admin/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,5 +1,12 @@
-kafka.brokers=${KAFKA_BROKERS}
-kafka.schema-registry-url=${KAFKA_SCHEMA_REGISTRY_URL}
+kafka.brokers=localhost:8081
```
Comment (pascal-airy) [inline backend/api/admin/src/main/resources/application.properties:1]: this might cause some problems :)

Comment (chrismatix) [reply backend/api/admin/src/main/resources/application.properties:1]: Ah forgot the debug file. Thanks! :D

**Review by pascal-airy — DISMISSED** [review]
🔥

**Review by lucapette — APPROVED** [review]
:shipit:

**Pull request merged**